README.md

```diff
@@ -84,6 +84,52 @@ By default the plugin will use the annotation `github.com/project-slug` and get
 - Add GitHub Insights plugin tab.
 - Show widgets about repository contributors, languages, readme and release at overview page.
 
+### Widgets
+
+#### Compliance Card
+```ts
+// packages/app/src/components/catalog/EntityPage.tsx
+import {
+EntityGithubInsightsComplianceCard
+} from '@roadiehq/backstage-plugin-github-insights';
+```
+![a preview of the compliance widget](docs/compliance-report-widget.png)
+
+#### Contributors Card
+```ts
+// packages/app/src/components/catalog/EntityPage.tsx
+import {
+EntityGithubInsightsContributorsCard
+} from '@roadiehq/backstage-plugin-github-insights';
+```
+![a preview of the contributors widget](docs/contributors-widget.png)
+
+#### Languages Card
+```ts
+// packages/app/src/components/catalog/EntityPage.tsx
+import {
+EntityGithubInsightsLanguagesCard
+} from '@roadiehq/backstage-plugin-github-insights';
+```
+![a preview of the languages widget](docs/languages-widget.png)
+
+#### ReadMeCard
+```ts
+// packages/app/src/components/catalog/EntityPage.tsx
+import {
+EntityGithubInsightsReadmeCard
+} from '@roadiehq/backstage-plugin-github-insights';
+```
+![a preview of the compliance widget](docs/readme-widget.png)
```

Comment:
It looks like you might have dropped the wrong image here. When I view your branch in the browser I see a screenshot of the About Card that comes with Backstage.

![Screenshot 2021-05-07 at 18 04 52](https://user-images.githubusercontent.com/562403/117484303-d2010180-af5e-11eb-82b0-8723310b5593.png)